### PR TITLE
Amigo

### PR DESCRIPTION
**Pull request by gianmarcomennecozzi**
Amigo structure re-built:
- new CSS
- NPM version and packages updaged
- Removed all useless file

### CODE REVIEW

File: store/myevent.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package store
```
Comment (mrturkmen06) [inline store/myevent.go:1]: I assume that this file is created for test purposes ?  If it is not written for test purposes why it is called `myevent.go`

File: store/team.go [hunk 2]
```diff
@@ -103,22 +90,36 @@ func (es *teamstore) GetTeamByToken(token string) (*haaukins.Team, error) {
...
+
 	es.emails[email] = t.ID()
 	es.teams[t.ID()] = t
 	es.m.Unlock()
 
 	return nil
 }
 
+//todo does it create a token or it save it in teamstore? Ahmet. I thinnk it saves it. so i can change hte name of this function
 func(es *teamstore) CreateTokenForTeam (token string, in *haaukins.Team) error {
```
Comment (mrturkmen06) [inline store/team.go:123]: Yeah, the name of the function does not fully represent what it is doing. It just saves the existing given token to teamstore.

File: store/myevent.go (hunk 1 above)
Comment (gianmarcomennecozzi) [reply store/myevent.go:1]: Yeah it is :)

**Comment by mrturkmen06** [comment]
`the_real_amigo` branch migrated to `develop-a`, we agreed that this merge will not gonna happen.

**Pull request closed without merging**